src/rpc/methods/eth.rs

```diff
@@ -1411,4 +1412,36 @@ mod test {
         let block = Block::new(true);
         assert_eq!(block.transactions_root, Hash::default());
     }
+
+    #[test]
+    fn test_tx_args_to_address() {
+        // None case
+        let msg = Message {
+            version: 0,
+            to: FilecoinAddress::ETHEREUM_ACCOUNT_MANAGER_ACTOR,
+            method_num: EAMMethod::CreateExternal as u64,
+            ..Message::default()
+        };
+
+        assert!(eth_tx_args_from_unsigned_eth_message(&msg)
+            .unwrap()
+            .to
+            .is_none());
+
+        // Some case
+        let msg = Message {
+            version: 0,
+            to: FilecoinAddress::from_str("f410fujiqghwwwr3z4kqlse3ihzyqipmiaavdqchxs2y").unwrap(),
+            method_num: EVMMethod::InvokeContract as u64,
+            ..Message::default()
+        };
+
+        assert_eq!(
+            eth_tx_args_from_unsigned_eth_message(&msg)
+                .unwrap()
+                .to
+                .unwrap(),
+            Address(H160::from_str("0xa251031ed6b4779e2a0b913683e71043d88002a3").unwrap())
+        );
```

Comment:
Perhaps this could be split into a separate test? Not a big deal.

Reply:
Done.